crates/rustc_codegen_spirv/src/lib.rs

```diff
@@ -52,24 +55,27 @@
     clippy::pub_enum_variant_names,
     clippy::ref_option_ref,
     clippy::rest_pat_in_fully_bound_structs,
+    clippy::string_add_assign,
+    clippy::string_add,
     clippy::string_to_string,
     clippy::suboptimal_flops,
     clippy::todo,
+    clippy::unimplemented,
     clippy::unnested_or_patterns,
     clippy::unused_self,
     clippy::verbose_file_reads,
     future_incompatible,
     nonstandard_style,
     rust_2018_idioms
 )]
-// END - Embark standard lints v0.2
+// END - Embark standard lints v0.3
 // crate-specific exceptions:
 #![allow(
     unsafe_code,                // still quite a bit of unsafe
     clippy::map_unwrap_or,      // TODO: test enabling
     clippy::match_on_vec_items, // TODO: test enabling
     clippy::enum_glob_use,
-    clippy::todo,               // still lots to implement :)
+    clippy::todo                // still lots to implement :)
```

Comment:
Huh, we use `todo!()`? I remember trying to use `unimplemented!()` and getting an error from a lint.

Reply:
yup plenty of `todo!` in the crate